### ISSUE TITLE
chestshop.othername.destroy.* not working

### ISSUE THREAD

**Issue by Blueme00** [ChestShop-authors/ChestShop-3#423]
ChestShop version 3.11 (build 213)

[config.zip](https://github.com/ChestShop-authors/ChestShop-3/files/6185902/config.zip)

Server Version: This server is running Paper version git-Paper-271 (MC: 1.16.4) (Implementing API version 1.16.4-R0.1-SNAPSHOT)

Server Logs: https://mcpaste.io/32d150b157f98b15

Granted Permissions: https://luckperms.net/treeview/pmY3bUjYEX

Plugins (23): AnimalPlague, AuthMe, Bookshelves, ChestShop, Companies, DeathCommands*, DelayedCommandTasker*, Invsee*, LuckPerms, ore-control, Orebfuscator, ProtocolLib, PwnPlantGrowth, RandomSpawn*, SilkSpawner, SnoozeBar, Sudo*, TheNewEconomy, Towny, TownyChat, TownyNameUpdater, UltimateChairs, Vault

What happened: The chestshop.othername.destroy.* permission node doesn't work.  Many players are unable to destroy signs of others when they purchase a towny plot.  I've given the permission to several different players and their groups to allow them to destroy other players' shops but have yet to make this work.  They are, however, able to destroy the chests themselves, but not the signs.

What I expected: For this permission node, when granted, to give players the ability to destroy other players' shops.

**Comment by Phoenix616** [comment]
You are running an outdated version. Update to the latest development build and see if the issue persists.

**Comment by Blueme00** [comment]
That's the one that downloaded from Spigot yesterday.  I made sure to update it before submitting this.

**Comment by Blueme00** [comment]
ChestShop's version is: 3.11 (build 213)

**Comment by Phoenix616** [comment]
Yes, that's the last release. Please update to the latest [development build](https://ci.minebench.de/job/ChestShop-3/) which contains all the changes made since then.

**Comment by Blueme00** [comment]
ChestShop's version is: 3.12-SNAPSHOT (build 283)

My players and I are no longer able to create shops.

**Comment by Phoenix616** [comment]
> ChestShop's version is: 3.12-SNAPSHOT (build 283)
> 
> My players and I are no longer able to create shops.

Please open a separate ticket for unrelated issues.
